packages/web3-eth-contract/src/contract.ts

```diff
@@ -126,19 +141,84 @@ export class Contract<Abi extends ContractAbi>
 	>
 	implements Web3EventEmitter<ContractEventEmitterInterface<Abi>>
 {
+	/**
+	 * The options `object` for the contract instance. `from`, `gas` and `gasPrice` are used as fallback values when sending transactions.
+	 *
+	 * ```ts
+	 * myContract.options;
+	 * > {
+	 *     address: '0x1234567890123456789012345678901234567891',
+	 *     jsonInterface: [...],
+	 *     from: '0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe',
+	 *     gasPrice: '10000000000000',
+	 *     gas: 1000000
+	 * }
+	 *
+	 * myContract.options.from = '0x1234567890123456789012345678901234567891'; // default from address
+	 * myContract.options.gasPrice = '20000000000000'; // default gas price in wei
+	 * myContract.options.gas = 5000000; // provide as fallback always 5M gas
+	 * ```
+	 */
 	public readonly options: ContractOptions;
 
+	/**
+	 * Can be used to set {@link Contract.defaultAccount} for all contracts.
```

Comment:
I think there should be more detail like:
This default address is used as the default "from" property, if no "from" property is specified in for the following methods:
new web3.eth.Contract() -> myContract.methods.myMethod().call()
new web3.eth.Contract() -> myContract.methods.myMethod().send()

Reply:
Yes all that detail should go under the documentation of the `Wbe3Context` property `defaultAccount`.

Reply:
lets add doc for these properties via web3context Web3ConfigOptions in this PR and update all @links of defaults in contract doc.

Reply:
We should add docs for those properties in separate PR concerning web-core and web3-common packages. And links still refers to contract instance as it inherits those properties.

Reply:
Exactly, plus if you are referring defaults from config in doc of contract that link should refer to actual def instead to referring to same property in contract.